sources/packages/backend/workflow/test/2022-2023/assessment-gateway/assessment-gateway.studentAppeal.e2e-spec.ts

```diff
@@ -28,15 +31,19 @@ describe(`E2E Test Workflow assessment gateway on student appeal for ${PROGRAM_Y
       ...createFakeSingleIndependentStudentData(),
     };
 
+    const workersMockedData = createWorkersMockedData([
+      createLoadAssessmentDataTaskMock({
+        assessmentConsolidatedData: assessmentConsolidatedData,
```

Comment:
We are ignoring the subprocess context here, which is great 👍

Reply:
Yes, I resolved this comment using the same 😉 
https://github.com/bcgov/SIMS/pull/1865#discussion_r1161962160